src/screens/Auth/LoginScreen.tsx

```diff
@@ -39,11 +42,12 @@ export const LoginScreen = (props: LoginScreenProps) => {
         message: null,
     });
     const [password, setPassword] = useState<PasswordState>({
-        value: 'password3',
+        value: 'password',
         valid: true,
         visible: false,
         message: null,
     });
+    const [pushToken, setPushToken] = useState(null);
```

Comment:
this doesn't seem to be used anywhere?

Reply:
yeah, its removed in my next branch

Reply:
you can update it here, and it won't cause any merge conflicts on your next branch